lib/driver.js

```diff
@@ -172,11 +171,11 @@ class XCUITestDriver extends BaseDriver {
 
     if (this.opts.browserName) {
       log.info('Safari test requested');
-      this.safari = true;      
+      this.safari = true;
       this.opts.app = undefined;
       this.opts.processArguments = this.opts.processArguments || {};
       if (this.isRealDevice()) {
```

Comment:
What is the need for this split, then? The only thing different, it seems, is the initial url.